src/components/Pursuance/PursuanceMenu.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Navbar, NavItem, Nav, ButtonGroup, Button, DropdownButton, MenuItem } from 'react-bootstrap';
+import { NavLink } from 'react-router-dom';
+import PursuanceMenuItem from './PursuanceMenuItem';
+import FaCheckSquareO from 'react-icons/lib/fa/check-square-o';
+import CommentsO from 'react-icons/lib/fa/comments-o';
+import PaperClip from 'react-icons/lib/fa/paperclip';
+import Group from 'react-icons/lib/fa/group';
+import Rocket from 'react-icons/lib/fa/rocket';
+import PlusCircle from 'react-icons/lib/fa/plus-circle';
+
+const PursuanceMenu = ({ pursuanceId }) => {
+  return (
+    <ButtonGroup vertical>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='New'
+          action='add'
+          icon={<PlusCircle size={28} />}
+        />
+      </Button>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='Tasks'
+          action='tasks'
+          icon={<FaCheckSquareO size={28} />}
+        />
+      </Button>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='Discuss'
+          action='discussion'
+          icon={<CommentsO size={28} />}
+        />
+      </Button>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='Files & Docs'
+          action='docs'
+          icon={<PaperClip size={28} />}
```

Comment:
@Spetastian When I see paperclip I think "attachment(s)" rather than files or documents, but I haven't asked users to compare this to an icon of a folder or a doc.  Think paperclip is better than http://fontawesome.io/icon/folder-open/ ?

Reply:
Especially given that we'll probably have a tree of folders containing files?

Reply:
Based on yesterdays discussion i agree, a folder is a much better icon.